did/utils.py

```diff
@@ -187,6 +187,27 @@ def shorted(text, width=MAX_WIDTH):
     return "\n".join(lines)
 
 
+def strtobool(val):
+    """
+    Convert a string representation of truth to true (1) or false (0).
+    Reimplemented following instructions at
+    https://peps.python.org/pep-0632/#migration-advice
```

Comment:
Could you please add note that the code is an exact copy from  https://github.com/pypa/distutils/blob/ee021a1c58b43607ccc75447159bd90f502c6bea/distutils/util.py#L340

That code is under MIT license while all other `did` code is GPL so I think we should make it explicit (not a lawyer though).

Reply:
Done, also added a copy of MIT license.